doc/CHANGELOG

```diff
@@ -221,6 +221,7 @@ KallistiOS version 2.1.0 -----------------------------------------------
 - DC  Added new driver for the SH4's UBC + high-level breakpoint API [FG]
 - DC  Add support for French AZERTY keyboards [PC]
 - DC  Increased the resolution of pvr_stats_t from milli to nanoseconds [FG]
+- *** Added support for modifying scheduler frequency at runtime [FG && PC]
```

Comment:
Since libudev was removed, is the attribution here still correct, or should it just be you, @gyrovorbis?.

Reply:
Updating.